README.md

```diff
@@ -1275,6 +1277,14 @@ extension MYOwnCardPartController: CardPartsLongPressGestureRecognizerDelegate {
 }
 ```
 
+# Dark Mode
+
+CardParts now supports iOS 13's Dark Mode!
+
+<p align="center">
+<img src="https://raw.githubusercontent.com/Intuit/CardParts/master/images/darkmode.png" width="300" alt="darkmode"/>
+</p>
```

Comment:
It seems that the imagee isnt able to be rendered 🤔

Reply:
@lkeude96 that’s because we haven’t merged to master yet :)